### DISCUSSION TITLE
New LTSP with "old" screen_01, screen_02...

### DISCUSSION THREAD

**Discussion by SpiritX86** [ltsp/ltsp#574] [Q&A]
Good morning, First of all, thank you for develop and maintain this project. It's a great job!

I've got a very simple LTSP5 server that's booting clients only with screen_01=shell and screen_02=xfreerdp.

Since few weeks ago I'm trying to do an new LTSP 2019 server on ubuntu mate 20.04 and I want to boot exactly the same and add "screen_03=chrome in mode kiosk".

The problem is that I can't do this and I've only managed to start a normal desktop every time, all that I'm trying inside ltsp.conf it's not working.

Can anyone help me please?

Thank you in advance!

**Comment by alkisg** [comment]
The new LTSP doesn't include support for SCREEN variables, nor does it have a kiosk mode.
It's up to the sysadmin to do these.

Here's a related discussion: https://github.com/ltsp/ltsp/discussions/514#discussioncomment-1258238

**Reply by HunterTrentLowe** [reply]
Is there any special reason that the screen variables were no supported? It is the missing piece to my setup

**Comment by alkisg** [comment]
The main reason is that no LTSP developer needed it, and noone contributed code for it.
What's your use case, and why isn't the method I proposed above easy enough for you?
